microapps/BasemapGalleryApp/app/src/main/java/com/arcgismaps/toolkit/basemapgalleryapp/screens/MainScreen.kt

```diff
@@ -0,0 +1,115 @@
+ *  limitations under the License.
+ *
+ */
+
+package com.arcgismaps.toolkit.basemapgalleryapp.screens
+
+import android.util.Log
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.BottomSheetScaffold
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.SegmentedButton
+import androidx.compose.material3.SegmentedButtonDefaults
+import androidx.compose.material3.SheetValue
+import androidx.compose.material3.SingleChoiceSegmentedButtonRow
+import androidx.compose.material3.Text
+import androidx.compose.material3.TopAppBar
+import androidx.compose.material3.rememberBottomSheetScaffoldState
+import androidx.compose.material3.rememberStandardBottomSheetState
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.arcgismaps.mapping.Basemap
+import com.arcgismaps.mapping.BasemapStyleInfo
+import com.arcgismaps.mapping.Item
+import com.arcgismaps.toolkit.basemapgallery.BasemapGallery
+import com.arcgismaps.toolkit.basemapgalleryapp.ViewModel
+import com.arcgismaps.toolkit.geoviewcompose.MapView
+
+/**
+ * The main screen of the application consisting of a [MapView] and a [BasemapGallery]. Clicking on
+ * an item in the gallery will set that basemap in the map view.
+ */
+@Composable
+@OptIn(ExperimentalMaterial3Api::class)
+fun MainScreen() {
+    val viewModel: ViewModel = viewModel()
+
+    val bottomSheetScaffoldState = rememberBottomSheetScaffoldState(
+        bottomSheetState = rememberStandardBottomSheetState(
+            initialValue = SheetValue.Expanded
+        )
+    )
+
+    val options = listOf("Basemap Styles", "Portal items")
+    var selectedBasemapSource by remember { mutableIntStateOf(0) }
+
+    BottomSheetScaffold(
+        sheetContent = {
+            SingleChoiceSegmentedButtonRow(modifier = Modifier.padding(10.dp).fillMaxWidth()) {
+                options.forEachIndexed { index, label ->
+                    SegmentedButton(
+                        shape = SegmentedButtonDefaults.itemShape(
+                            index = index,
+                            count = options.size
+                        ),
+                        onClick = { selectedBasemapSource = index },
+                        selected = index == selectedBasemapSource,
+                        label = { Text(label) }
+                    )
+                }
+            }
+
+            BasemapGallery(modifier = Modifier.fillMaxHeight(fraction = 0.5f),
+                basemapGalleryItems = when (selectedBasemapSource) {
```

Comment:
Nitpick - I find it more readable if each parameter goes on a new line when many parameters are passed. Like:
 ```suggestion
            BasemapGallery(
                modifier = Modifier.fillMaxHeight(fraction = 0.5f),
                basemapGalleryItems = when (selectedBasemapSource) {
```

Applies in other places but only mention it here.